.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''
+assignees: timmydoza
```

Comment:
remove assignees

Reply:
add label "feature request"

Reply:
Added label 'feature' to correspond with the label:
![image](https://user-images.githubusercontent.com/12685223/74545563-51438900-4f06-11ea-92fc-3f9de690b3c6.png)